Beiwe/Recline/Recline.swift

```diff
@@ -31,175 +29,158 @@ class Recline {
             self._open(dbName)
         }
     }
-    
+
     func _open(_ dbName: String = "default") {
```

Comment:
It looks like you substantially simplified this load function, I don't know what it was doing before, where is this change coming from.
(if this works with existing app installations, cool.  If not, not cool.)

Reply:
you said
- I updated Recline implementation to use collection API required in the new Couchbase version. I found no error after refactoring, but you might want to test this further.
can you direct me to documentation?

Reply:
Here's the documentation I referred to when I did the refactoring: https://docs.couchbase.com/couchbase-lite/current/swift/quickstart.html

To my knowledge, Couchbase view often used to create database view and can be useful if you want to implement map reduce calculation https://en.wikipedia.org/wiki/MapReduce

Since the use of Couchbase in this app is simpler than that, I remove the Couchbase view creation.